gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/main/java/io/gravitee/gateway/jupiter/reactor/handler/context/AbstractExecutionContext.java

```diff
@@ -60,7 +66,7 @@ public RS response() {
 
     @Override
     public Completable interrupt() {
-        return Completable.error(new InterruptionException());
+        return messagesInterruption();
```

Comment:
`messageInterruption` without the '**s**' ?

Reply:
this is used either by  interruptMessageS or interruptMessage. i would say both could work.